### DISCUSSION TITLE
How to create a custom responsive version of Konsta UI from mobile view to desktop view?

### DISCUSSION THREAD

**Discussion by amazingnerd** [konstaui/konsta#109] [Q&A]
Hello everyone,
Is it possible to create a custom responsive version of Konsta UI from mobile view to desktop view? And if yes, how can we do this? Thank you a lot in advance!

**Comment by jkhaui** [comment]
I have this idea that it'd be pretty sweet to create a component library that renders radix/shadcn components on desktop and the Konsta variant on mobile.

The components would share a common interface (most likely extending Konsta's props with a `desktop` variant, but this is just an implementation detail).

I've begun working on a proof of concept for this and it looks promising. Getting the tailwind config set up in a way that shares common values like the runtime colour space, in a way that is still easily customisable, was a pain - but it looks like the end result would justify the effort.

If anyone is interested in collaborating on this let me know!

**Reply by alextwothousand** [reply]
Sounds interesting - let us know if you got there in the end!
